bin/build.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Prevent jenkins from immediately killing the script when a step fails, allowing us to notify github:
+set +e
+
+BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+BASEDIR="${BIN_DIR}/.."
+cd $BIN_DIR
+
+# Env config:
+source $BASEDIR/manifest.sh
+if [ -n "$ghprbActualCommit" ]; then
+    # always overrides default GIT_COMMIT:
+    GIT_COMMIT=$ghprbActualCommit
+fi
+if [ -z "$GIT_COMMIT" ]; then
+    # Commit not explicitly provided by CI. Fetch directly from Git:
+    GIT_COMMIT="$(git rev-parse HEAD)"
+fi
+if [ -z "$GIT_COMMIT" ]; then
+    echo "Unable to determine git commit. Giving up."
+    exit 1
+fi
+if [ -z "$DOCKER_IMAGE" ]; then
+    export DOCKER_IMAGE="mesosphere/spark-dev:$GIT_COMMIT"
+fi
+
+# Grab dcos-commons build/release tools:
+rm -rf dcos-commons-tools/ && curl https://infinity-artifacts.s3.amazonaws.com/dcos-commons-tools.tgz | tar xz
```

Comment:
To receive updates to this repo, can we add this as a source repo to the jenkins build, instead of pulling a tarball?